data/mods.json

```diff
@@ -494,5 +494,27 @@
     "size": 2949,
     "url": "https://github.com/Entity-y/OG_Drugs/archive/main.zip",
     "homepage": "https://discourse.cataclysmdda.org/t/original-drug-names/25335"
+  },
+  {
+    "type": "direct_download",
+    "ident": [ "Kenan2000s_modpack" ],
+    "name": "Kenan2000's Modpack",
+    "description": "Kenan's personal modpack for Cataclysm - Dark Days Ahead for LATEST experimental versions.",

```

Comment:
Link below indicates it is for stable 0.E-3.

Reply:
The description is copied verbatim from the repo, but I guess I could change it to be more accurate.